src/applets/clock/ClockApplet.vala

```diff
@@ -206,6 +138,17 @@ public class ClockApplet : Budgie.Applet {
 			return Gdk.EVENT_STOP;
 		});
 
+		
+
+		// make sure every setting is ready
+		this.update_setting("clock-show-date");
+		this.update_setting("clock-show-seconds");
+		this.update_setting("clock-use-24-hour-time");
+		this.update_setting("clock-use-custom-format");
+		this.update_setting("clock-custom-format");
+		this.update_setting("clock-use-custom-timezone");
+		this.update_setting("clock-custom-timezone");
```

Comment:
The `clock-` prefix is a bit redundant seeing as it is within each clock applet schema, so that prefix can be removed.